lib/ansible/modules/cloud/docker/_docker.py

```diff
@@ -25,7 +25,7 @@
 description:
   - This is the original Ansible module for managing the Docker container life cycle.
   - NOTE - Additional and newer modules are available. For the latest on orchestrating containers with Ansible
-    visit our Getting Started with Docker Guide at U(https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/guide_docker.rst).
+    visit our Getting Started with Docker Guide at U(https://docs.ansible.com/ansible/latest/scenario_guides/guide_docker.html).
```

Comment:
This needs to be fixed in the latest branch in which it still exists (stable-2.6) and then backported to older branches once the fix is merged.

Also, links to our docs shouldn't be using an URL. I believe this is the correct syntax:

```suggestion
    visit our L(Getting Started with Docker Guide,.../scenario_guides/guide_docker.html).
```

@acozine, can you confirm?

Reply:
@mattclay confirmed - the correct formats are documented on https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation.